community/cecilialevenstein.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
# Oi! Oi sou a Ciça!


## 🚀 About Me
Migrando para a área de T.I.


## 🔗 Links
[![portfolio](https://img.shields.io/badge/my_portfolio-000?style=for-the-badge&logo=ko-fi&logoColor=white)](https://abovewonderland.weebly.com/)

[![linkedin](https://img.shields.io/badge/-Instagram-%23E4405F?style=for-the-badge&logo=instagram&logoColor=white)](https://www.linkedin.com/in/cecilialevenstein)

[![instagram](https://img.shields.io/badge/linkedin-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.instagram.com/cilevenstein)

[![gmail](https://img.shields.io/badge/-Gmail-%23333?style=for-the-badge&logo=gmail&logoColor=white)](mailto:c.helenalevenstein@gmail.com)


## Estatísticas no GitHub
![cecilialevenstein's Stats](https://github-readme-stats.vercel.app/api?username=cecilialevenstein&theme=tokyonight&show_icons=true&hide_border=true&count_private=true)![cecilialevenstein's Streak](https://github-readme-streak-stats.herokuapp.com/?user=cecilialevenstein&theme=tokyonight&hide_border=true)
![cecilialevenstein's Top Languages](https://github-readme-stats.vercel.app/api/top-langs/?username=cecilialevenstein&theme=tokyonight&show_icons=true&hide_border=true&layout=compact)